cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -99,8 +99,14 @@ if("@MSVC@")
   set(bootstrap_cmd "bootstrap.bat")
 elseif("@MINGW@")
   # Test scenario: MinGW installed in system, Visual Studio - not
-  set(install_cmd "b2")
-  set(bootstrap_cmd "bootstrap.bat" "gcc")
+  if("${CMAKE_CROSSCOMPILING}")
```

Comment:
We can use both `${CMAKE_CROSSCOMPILING}` or `@CMAKE_CROSSCOMPILING@` however when you mix two it look inconsistent. Let's use `@CMAKE_CROSSCOMPILING@` everywhere.